### DISCUSSION TITLE
Bug Report with Cp2k_aimdoutput

### DISCUSSION THREAD

**Discussion by xiangyyu** [deepmodeling/dpdata#342] [General]
Hi,

I would like to report a bug I found today. I apply cp2k AIMD trajectories as the dataset. I used to use the older version cp2k 5.1, and it worked well. Recently I switched to a newer version of cp2k 8.2, and when I tried to dump the data into 'deepmd/npy', it reported an error: nframes = data['cells'].shape[0] AttributeError: 'list' object has no attribute 'shape'.

After troubleshooting, I found that it is because the newer version cp2k will add two more lines of cell volume and one more line of cell lengths to the cell information section, which messes up the format.
This is the output from the newer version cp2k 8.2:
`MD_INI| MD initialization`
`MD_INI| Potential energy [hartree]                          -0.165294216820E+03`
`MD_INI| Kinetic energy [hartree]                             0.213760035672E-01`
`MD_INI| Temperature [K]                                              300.000000`
`MD_INI| Cell volume [bohr^3]                                 5.398667666472E+04`
`MD_INI| Cell volume [ang^3]                                  8.000000000000E+03`
`MD_INI| Cell lengths [bohr]      3.77945227E+01  3.77945227E+01  3.77945227E+01`
`MD_INI| Cell lengths [ang]       2.00000000E+01  2.00000000E+01  2.00000000E+01`
`MD_INI| Cell angles [deg]        9.00000000E+01  9.00000000E+01  9.00000000E+01`

This is the output from the older cp2k 5.1:
`INITIAL POTENTIAL ENERGY[hartree]     =                     -0.131804409525E+03`
`INITIAL KINETIC ENERGY[hartree]       =                      0.142506690448E-01`
`INITIAL TEMPERATURE[K]                =                                 300.000`
`INITIAL VOLUME[bohr^3]                =                      0.105442727861E+06`
`INITIAL CELL LNTHS[bohr]   =      0.4724315E+02   0.4724315E+02   0.4724315E+02`
`INITIAL CELL ANGLS[deg]    =      0.9000000E+02   0.9000000E+02   0.9000000E+02`

I manually deleted these additional lines and dpdata worked again. 

I am using dpdata 0.2.5. I went to the Github homepage of dpdata, and I did not see this bug involved in the releases of the newer version. That is the reason why I would like to reach out to you and report this bug.

Thanks!

**Comment by HuangJiameng** [comment]
`cp2k_aimdoutput` only support cp2k 7.1 currently. Anyway, thanks for your report.
